transport.go

```diff
@@ -160,7 +160,7 @@ type Transport struct {
 
 // Listen starts listening for incoming QUIC connections.
 // There can only be a single listener on any net.PacketConn.
-// Listen may only be called again after the current Listener was closed.
+// Listen may only be called again after the current [Listener] was closed.
```

Comment:
This applies to both `Listener` and `EarlyListener`.
```suggestion
// Listen may only be called again after the current listener was closed.
```